syncer/dml_test.go

```diff
@@ -44,3 +44,63 @@ func (s *testSyncerSuite) TestCastUnsigned(c *C) {
 		c.Assert(obtained, Equals, cs.expected)
 	}
 }
+
+func (s *testSyncerSuite) TestGenColumnPlaceholders(c *C) {
+	placeholderStr := genColumnPlaceholders(3)
+	c.Assert(placeholderStr, Equals, "?,?,?")
```

Comment:
I think you can also test with **1** `?` here.

Reply:
done